src/utils/conversions.ts

```diff
@@ -0,0 +1,17 @@
+export const vestsToHp = (vests, hivePerMVests) => {
+  if (!vests || !hivePerMVests) {
+    return 0;
+  }
+
+  return (vests / 1e6) * hivePerMVests;
+};
+
+export const vestsToRshares = (vests:number, votingPower:number, weight:number) => {
+  if (!vests || !votingPower || !weight) {
+    return 0;
+  }
+
+  const finalVest = vests * 1e6;
+  const power = (votingPower * weight / 10000) / 50
+  return (power * finalVest / 1000) 
```

Comment:
@noumantahir here one 0 missing, no? used to be `/1e4`

Reply:
Will compare shortly

Reply:
I double checked, only thing I could find varying is weight, as I am used weight calculation from previous formula as not information given in their documentation.

`  const weight = sliderValue * 10000;`